Packages/com.unity.inputsystem/InputSystem/Plugins/HID/HID.cs

```diff
@@ -600,7 +600,7 @@ internal string DetermineName()
                             return "hat";
                         var text = ((GenericDesktop)usage).ToString();
                         // Lower-case first letter.
-                        text = char.ToLowerInvariant(text[0]) + text.Substring(1);
+                        text = char.ToUpperInvariant(text[0]) + text.Substring(1).ToUpperInvariant();
```

Comment:
I think based on the comment this part was probably correct before. (Just wanted the first character to be uppercase).

Reply:
your right, there I was too zealously changing all ToLower